src/Script/Composer/Composer.php

```diff
@@ -6,11 +6,23 @@
 
 class Composer
 {
-    public static function removeUnusedServices(
-        Event      $event,
-        Filesystem $filesystem = null
-    )
+
+    public static function removeUnusedServicesInDevOnly(Event $event, Filesystem $filesystem = null)
```

Comment:
Would you please rename this to `removeUnusedServicesInDev` instead of `removeUnusedServicesInDevOnly`?

Thanks!